controllers/aliases.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const config = require('config');
+const { aliasFor, vanity, programmeRedirect } = require('./route-types');
+
+const anchors = config.get('anchors');
+
+/**
+ * Legacy Redirects
+ */
+const legacyRedirects = [
+    aliasFor('/funding/programmes', '/Home/Funding/Funding*Finder'),
+    aliasFor('/funding/programmes?location=scotland', '/funding/scotland-portfolio'),
+    aliasFor(
+        '/funding/programmes/national-lottery-awards-for-all-england',
+        '/england/global-content/programmes/england/awards-for-all-england'
+    ),
+
+    // Migrated Programme Pages [LIVE]
+    programmeRedirect('england/awards-for-all-england', 'national-lottery-awards-for-all-england'),
+    programmeRedirect('england/reaching-communities-england', 'reaching-communities-england'),
+    programmeRedirect('northern-ireland/awards-for-all-northern-ireland', 'awards-for-all-northern-ireland'),
+    programmeRedirect('scotland/awards-for-all-scotland', 'national-lottery-awards-for-all-scotland'),
+    programmeRedirect('scotland/grants-for-community-led-activity', 'grants-for-community-led-activity'),
+    programmeRedirect('scotland/grants-for-improving-lives', 'grants-for-improving-lives'),
+    programmeRedirect('wales/people-and-places-medium-grants', 'people-and-places-medium-grants'),
+    programmeRedirect('wales/people-and-places-large-grants', 'people-and-places-large-grants'),
+    programmeRedirect('uk-wide/uk-portfolio', 'awards-from-the-uk-portfolio'),
+    programmeRedirect('uk-wide/coastal-communities', 'coastal-communities-fund'),
+    programmeRedirect('uk-wide/lottery-funding', 'other-lottery-funders'),
+    programmeRedirect('northern-ireland/people-and-communities', 'people-and-communities'),
+    programmeRedirect('wales/awards-for-all-wales', 'national-lottery-awards-for-all-wales'),
+    programmeRedirect('england/building-better-opportunities', 'building-better-opportunities'),
+
+    // Migrated Programme Pages [DRAFT]
+    programmeRedirect('england/parks-for-people', 'parks-for-people', false),
+    programmeRedirect('scotland/community-assets', 'community-assets', false),
+    programmeRedirect('uk-wide/east-africa-disability-fund', 'east-africa-disability-fund', false),
+    programmeRedirect('scotland/our-place', 'our-place', false),
+    programmeRedirect('scotland/scottish-land-fund', 'scottish-land-fund', false),
```

Comment:
Might have a conflict here with https://github.com/biglotteryfund/blf-alpha/pull/646/files#diff-52503e8aa50e6bc9ca23de0c1dbab19fR273

Reply:
Happy to reapply them, I did this to myself.